newIDE/app/src/Utils/Object.js

```diff
@@ -0,0 +1,396 @@
+﻿/**
```

Comment:
There are a lot of functions here, but only a few are used. Could we remove all the ones that are not used?

Reply:
I know that you've been putting a lot of work in this, but I prefer that we entirely remove `Object.js`. I'm pretty sure we can implement the whole feature in a simpler way without them.
We'll add an utility function if really really needed - in which case it must be typed and limited to the file where it's used.